packages/xo-server-audit/src/index.js

```diff
@@ -219,4 +240,13 @@ class AuditXoPlugin {
   }
 }
 
+AuditXoPlugin.prototype._getRecordsStream = asyncIteratorToStream(
+  async function*(id) {
+    for await (const record of this._auditCore.getFrom(id)) {
+      yield JSON.stringify(record)
+      yield '\n'
+    }
+  }
+)
```

Comment:
Did you test it?

Because I wrote the code directly on GitHub without any tests :sweat_smile:

Reply:
yes i test it :+1: